cves/CVE-2014-0117.yml

```diff
@@ -219,4 +246,9 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    I think there was a simple error of not thorough enough tests. Having an empty
+    Connection Header seems not like an over-complicated input to test. I understand
+    it was more niche as it was while use of proxy, but proxy servers themselves are not
+    niche at all. I think they just need to extend their unit tests to modules as well.
```

Comment:
Were there not any mistakes with the implementation itself? Is there something the developer should've put more thought into or considered? I feel there might be something more to this than just testing.